### ISSUE TITLE
Technology Research (Map APIs)

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#15]
## Acceptance criteria
* Evaluated different map technologies
* Evaluated different licenses for bill of material
* Decision based on project requirements

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by dancingsushii** [comment]
To be done: paste it to Wiki @CelineMP

**Comment by CelineMP** [comment]
https://github.com/amosproj/amos2024ss04-building-information-enhancer/wiki/Frontend-Decisions#map-technology
